plugin.py

```diff
@@ -144,6 +146,51 @@ def log_solar_charger(
         print(json.dumps(delta))
         sys.stdout.flush()
 
+    def log_dcdc_converter(
+        self,
+        bl_device: BLEDevice,
+        cfg_device: ConfiguredDevice,
+        data: DcDcConverter,
+        id_: str,
+    ):
+        delta = {
+            "updates": [
+                {
+                    "source": {
+                        "label": "Victron",
+                        "type": "Bluetooth",
+                        "src": bl_device.address,
+                    },
+                    "timestamp": datetime.datetime.utcnow().isoformat() + "Z",
+                    "values": [
+                        {
+                            "path": f"electrical.converters.{id_}.chargingMode",
+                            "value": data.get_charge_state().name.lower(),
+                        },
+                        {
+                            "path": f"electrical.converters.{id_}.chargerError",
+                            "value": data.get_charger_error().name.lower(),
+                        },
+                        {
+                            "path": f"electrical.converters.{id_}.input.voltage",
+                            "value": data.get_input_voltage(),
+                        },
+                        {
+                            "path": f"electrical.converters.{id_}.output.voltage",
+                            "value": data.get_output_voltage(),
+                        },
+                        {
+                            "path": f"electrical.converters.{id_}.chargingMode.offReason",
+                            "value": data.get_off_reason().name.lower(),
+                        },
+                    ],
+                },
+            ],
+        }
+        logger.info(delta)
+        print(json.dumps(delta))
+        sys.stdout.flush()
```

Comment:
Clearly I need to do some refactoring and clean up this common stuff...

Reply:
looks good, just merged your changes into the branch